package.json

```diff
@@ -60,8 +59,6 @@
     "gzip-size": "^3.0.0",
     "inquirer": "^0.11.0",
     "jsdoc": "^3.4.0",
-    "jshint": "^2.9.1",
```

Comment:
It works without jshint listed here because ckeditor5-dev-utils requires it, so it gets installed in `ckeditor5` because it's the topmost package. But it may not be reliable unfortunately. If there's some conflict with some other package jshint may be installed in other place and it's going to break.

So, we could live without jshint being listed here and there's a high chance that nothing will break, but it's unsafe.